website/blog/2016-03-11-blog-post.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Blog Title
```

Comment:
I don't think we should add a bunch of fake blog posts, they're likely to be forgotten.

When we get blog posts we can add blog posts, but I don't see the point in adding fake ones now.

Reply:
I think we can safely point people at the docusaurus documentation and they can add what they need at their discretion. We do not know when we are going to start the blog etc. so it would be better to not start with a bunch of things we will have to track for deletion later.

Reply:
I'm curious, have you read the docusaurus documentation?  Are you going to be working in the documentation?  At this point I assume Jason and myself will be in here the most.  So, I want to keep this.